common/supplemental/plurals.xml

```diff
@@ -28,8 +28,8 @@ For terms of use, see http://www.unicode.org/copyright.html
         </pluralRules>
         <pluralRules locales="fr">
             <pluralRule count="one">i = 0,1 @integer 0, 1 @decimal 0.0~1.5</pluralRule>
-			<pluralRule count="many">e = 0 and i % 1000000 = 0 and v = 0 or e != 0 .. 5 @integer 1000000</pluralRule>
-            <pluralRule count="other"> @integer 2~17, 100, 1000, 10000, 100000, 1000000, … @decimal 2.0~3.5, 10.0, 100.0, 1000.0, 10000.0, 100000.0, 1000000.0, …</pluralRule>
+            <pluralRule count="many">e = 0 and i != 0 and i % 1000000 = 0 and v = 0 or e != 0 .. 5 @integer 1000000</pluralRule>
```

Comment:
I'm not a plural rules expert, but doesn't the "or e != 0 .. 5" mean that this rule matches all numbers with e >= 6?  Like all of 1M, 1.1M, 1.2M?

Reply:
Yes. I suspect we'll need to tweak this further.